src/AvailabilityCheckerInterface.php

```diff
@@ -29,9 +29,8 @@ public function applies(PurchasableEntityInterface $entity);
    * @param \Drupal\commerce\Context $context
    *   The context.
    *
-   * @return bool|null
-   *   TRUE if the entity is available, FALSE if it's unavailable,
-   *   or NULL if it has no opinion.
+   * @return \Drupal\commerce\AvailabilityResponseInterface
```

Comment:
(I ended up just moving all the AvailabilityResponse* classes into the root ```commerce``` namespace.)